lib/perl/Genome/Model/PhenotypeCorrelation/Command/CaseControl/Unrelated.pm

```diff
@@ -297,7 +297,8 @@ sub _create_workflow {
                 work_dir => $vep_work_directory,
                 ensembl_annotation_build => $ensembl_annotation_build,
                 log_dir => $log_dir,
-
+                analysis_build => $self->build,
+                analysis_project => $self->build->model->analysis_projects,
```

Comment:
RHS can be a list.  Assigning to a scalar would trigger an exception if that's encountered.  As is it could add garbage to the inputs.